autotest/gcore/gdal_api_proxy.py

```diff
@@ -641,6 +641,8 @@ def gdal_api_proxy_sub_clean():
         import time
 
         p = None
+        # set port to None in case the loop has no iterations
+        port = None
```

Comment:
does not make sense. The loop has necessary one iteration. The comment should probably be modified or a linter exception added